### PR TITLE
Added support for Kirby CLI

### PR DESCRIPTION
**Pull request by mxub**
I want to use the Kirby CLI whenever I can so I added the package.json

 You should edit the `author` and `license` the way you want it

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "map",
+  "description": "Kirby Map Field",
+  "author": "August Miller",
+  "version": "1.1.0",
+  "type": "kirby-plugin",
+  "license": "addMe"
```
Comment (AugustMiller) [inline package.json]: Mind updating this to include the recently added MIT license? Not sure what the format of the attribute should be.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "name": "map",
```
Comment (AugustMiller) [inline package.json:2]: Does this name have any influence on the directory that the plugin is installed to? My understanding is that this is important in how the primary plugin definition is loaded.

**Review by AugustMiller — COMMENTED** [review]
Thanks for submitting! Care to take a look at my comments, and update as necessary? ✌️

File: package.json (hunk 2 above)
Comment (mxub) [reply package.json:2]: the `name` is the name of the folder created in the plugin-folder

File: package.json (hunk 1 above)
Comment (mxub) [reply package.json]: done with be576d0

**Pull request merged**